README.md

```diff
@@ -1 +1,131 @@
-# rfq-marketplace
\ No newline at end of file
+# rfq-marketplace
+
+rfq-marketplace is a contract that provides a token exchange platform.
+
+## Prerequisites
+
+- Node.js
+- NPM
+
+## Deploy the contract
+
+The first step, clone the project and install the dependency.
+```
+git clone https://github.com/second-state/rfq-marketplace.git
+cd rfq-marketplace
+npm install
+```
+You can edit the `hardhat.config.js` to choose which network and account to deploy the contract.
+In the example, we use CyberMiles chain to demo.
+
+Next step, run the following command to run the deploy script.
+```
+npx hardhat run script/deploy.js --network cyber
+```
+
+Then you can see the contract address.
+```
+Deployed to 0xAe7FEF0Afdaf803E2665Ec6D83D8126D97dFcbFE
+```
+
+## Deploy web service on flows.network
+
+To interact with rfq-marketplace contract, we will use [flows.network](https://flows.network/), a serverless platform that makes deploying your own app quick and easy in just three steps.
+
+### Prerequisite
+
+You will need a wallet private key. If you do not already have one, use [Metamask](https://metamask.io/) to create it.
+
+#### Fork this repo and write your own code
+
+Fork [this repo](https://github.com/second-state/rfq-marketplace.git). 
+
+#### Deploy the code on flows.network
+
+1. Sign up for an account for deploying flows on [flows.network](https://flows.network/). It's free.
+2. Click on the "Create a Flow" button to start deploying the web service.
+3. Authenticate the [flows.network](https://flows.network/) to access the `rfq-marketplace` repo you just forked. 
+
+4. Click on the Advanced text and you will see more settings including branch and environment variables. The function code is stored in `flows` folder, you need to change Directory to `/flows`.
+ In this example, we have four variables to fill in, `PRIVATE_KEY` is the wallet private key, and `CONTRACT_ADDRESS` is rfq-market contract [address]( #deploy-the-contract).
```

Comment:
Whose private key is this? My guess is that it is the wallet that holds both token A and B, right? I am thinking maybe we should pass private keys as call parameters for related flow functions. That means the flow functions should use POST so that the private keys can be encrypted by the HTTPS.